src/core/execute.c

```diff
@@ -5164,6 +5170,18 @@ static void strv_fprintf(FILE *f, char **l) {
                 fprintf(f, " %s", *g);
 }
 
+static void strv_dump(FILE* f, const char *prefix, const char *name, char **strv) {
+        assert(f);
+        assert(prefix);
+        assert(name);
+
+        if (!strv_isempty(strv)) {
```

Comment:
in the interest of keeping indentation levels minimal I'd reverse this check and do a `return` here. i.e. return early is a good thing.